<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateCompanyWindow</class>
 <widget class="QMainWindow" name="UpdateCompanyWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KB Roadways - Update Company</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../__Assets/Logos/CompanyLogo.png</normaloff>../__Assets/Logos/CompanyLogo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblLogo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>171</width>
      <height>161</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../__Assets/Logos/CompanyLogo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="lwCompanyList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>261</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblContactNo">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Contact No.</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCompanyAddress">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Company Address</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leContactNo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="teCompanyAddress">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>541</width>
      <height>121</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="teNotes">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>350</y>
      <width>541</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblCompanyName">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Company Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNotes">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>310</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Notes (If any)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCompanyName">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDownloadAll">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDownloadSelected">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Download Selected</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUpdate">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDelete">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Constantia</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lwCompanyList</tabstop>
  <tabstop>btnDownloadSelected</tabstop>
  <tabstop>btnDownloadAll</tabstop>
  <tabstop>btnUpdate</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>leCompanyName</tabstop>
  <tabstop>leContactNo</tabstop>
  <tabstop>teCompanyAddress</tabstop>
  <tabstop>teNotes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
